helm-chart/kuberay-operator/values.yaml

```diff
@@ -35,6 +35,14 @@ resources:
   #   cpu: 100m
   #   memory: 512Mi
 
+logging:
+  # log encoder to use for stdout (one of 'json' or 'console')
+  stdoutEncoder: "console"
+  # log encoder to use for file logging (one of 'json' or 'console')
+  fileEncoder: "json"
+  # file path to store logs
+  file: ""
```

Comment:
Can you please split the `file` up into baseDir and fileName? E.g.
```
baseDir: /opt/ray/log
fileName: operator.log
```
that enables users to use a kubernetes volume as the baseDir and share the log folder between the operator and sidecars. That pattern is very useful when the logs are processed by a sidecar container. Then in the deployment.yaml, you could use 
```
{{- $argList = append $argList (printf "%s/%s" .Values.logging.baseDir .Values.logging.fileName) -}}
```

Reply:
> This enables users to use a kubernetes volume for the baseDir and share the log folder between the operator and sidecars. E.g.

@marton-bod references to the volume mounts would be used by the `ray-cluster` helm chart and not the `kuberay-operator` helm chart right? Unless in your use-caes you use the same values.yaml for both the operator and ray clusters

Reply:
(agree that we should add `.Values.logging.baseDir` in the ray-cluster helm chart)

Reply:
I was referring to the `kuberay-operator` only. The goal is to be able to package the operator's container together with a log forwarder sidecar container into the same pod, where both containers share the same volume mount (mounted under `.Values.logging.baseDir` in both). That allows the log forwarder sidecar to watch any operator's log files in the baseDir and send them to a log server for processing.

Reply:
Ah I see now -- it's still confusing to have `.Values.logging.baseDir` if the helm chart template doesn't reference a volume mount for the logs like you shared in https://github.com/ray-project/kuberay/pull/1912#discussion_r1481278324. 

Do you think we should add a default volume mount for the logs using emptyDir?

Reply:
> Do you think we should add a default emptyDir volume with the volume mount referencing .Values.logging.baseDir?

That's a great idea!

Reply:
Added baseDir and fileName and the emptyDir mount